components/camel-xmlsecurity/src/test/java/org/apache/camel/dataformat/xmlsecurity/EncryptionAlgorithmTest.java

```diff
@@ -197,6 +193,7 @@ public void configure() {
 
     @Test
     public void testAES256GCM() throws Exception {
+        assumeTrue(TestHelper.UNRESTRICTED_POLICIES_INSTALLED, "Test preconditions failed: UNRESTRICTED_POLICIES_INSTALLED="+TestHelper.UNRESTRICTED_POLICIES_INSTALLED);
         if (!TestHelper.UNRESTRICTED_POLICIES_INSTALLED) {
             return;
         }
```

Comment:
remove me

Reply:
Missing fix for https://github.com/apache/camel/blob/ea4bf39a9d6dd35eebe1c2af60e8e05b0a4fce02/components/camel-xmlsecurity/src/test/java/org/apache/camel/dataformat/xmlsecurity/EncryptionAlgorithmTest.java#L297-L299 and https://github.com/apache/camel/blob/ea4bf39a9d6dd35eebe1c2af60e8e05b0a4fce02/components/camel-xmlsecurity/src/test/java/org/apache/camel/dataformat/xmlsecurity/EncryptionAlgorithmTest.java#L325-L327